### PR TITLE
Run up prometheus in GKE too.

### PR DESCRIPTION
**Pull request by AlCutter**
Issue #405

### CODE REVIEW

File: cloud/prometheus/prometheus.conf [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Global settings and defaults.
+global {
+  scrape_interval: "10s"
+  evaluation_interval: "10s"
```
Comment (pphaneuf) [inline cloud/prometheus/prometheus.conf]: That's pretty hot, no? Or am I just used to, er, a different scale? :smiley:

File: cloud/prometheus/prometheus.conf [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  labels {
+    label {
+      name: "service"
+      value: "super_duper"
```
Comment (pphaneuf) [inline cloud/prometheus/prometheus.conf]: Shouldn't that be something more like "ct-server" or "ct_server"?

Comment (pphaneuf) [inline cloud/prometheus/prometheus.conf]: It's not like you can monitor "master", anyway... :wink:

**Comment by pphaneuf** [comment]
LGTM, my comments are possibly irrelevant, so I'll let you decide.

File: cloud/prometheus/prometheus.conf (hunk 1 above)
Comment (AlCutter) [inline cloud/prometheus/prometheus.conf]: Might be a bit hot for long-term production use, but kinda handy for now while it's barely working :)
We can certainly crank it down later on once it's all settled in.

Comment (pphaneuf) [inline cloud/prometheus/prometheus.conf]: SGTM

File: cloud/prometheus/prometheus.conf (hunk 2 above)
Comment (AlCutter) [inline cloud/prometheus/prometheus.conf]: Well, it's the overall service [i.e. all the jobs get that label too], so and "...server" isn't quite right, but I've changed it to "ct" for now.

**Comment by pphaneuf** [comment]
All right, fixed a few things I missed, too, looks even better than what I can tell. :wink:

**Pull request merged**